litmus-portal/frontend/src/components/WorkflowCluster/styles.ts

```diff
@@ -44,7 +44,12 @@ const useStyles = makeStyles((theme: Theme) => ({
   button: {
     marginLeft: 167,
     textAlign: 'left',
-    marginTop: 64,
+    marginTop: 67,
+  },
+  buttonOutlined:{
+    marginLeft: 359,
+      marginTop: -25,
+      textAlign: 'left',
   },
   or: {
     marginLeft: 347,
```

Comment:
same here

Reply:
Already changes are made @Saranya-jena  it's not updated commit